src/components/Filters/Filters.tsx

```diff
@@ -6,22 +6,22 @@ import styled from 'styled-components';
 import { FlexContainer } from '../FlexContainer';
 import { FilterRow } from './FilterRow';
 import { BottomBar } from './BottomBar';
-import { DataPointPropTypes, Filter, FiltersProps } from './Filters.types';
-import { DateRangePickerPropTypes } from './inputs/DateRangePicker/DateRangePicker.types';
+import { FieldPropTypes, Filter, FiltersProps } from './Filters.types';
+import { DateRangePickerPropTypes } from './components/DateRangePicker/DateRangePicker.types';
 import { Operators } from './Filters.enums';
 
-const generateId = ({ operator, dataPoint, condition }, index) =>
-  `${operator}-${dataPoint}-${condition}-${index}`;
+const generateId = ({ operator, field, condition }, index) =>
+  `${operator}-${field}-${condition}-${index}`;
 
-const getDefaultCondition = (dataPoints) =>
-  dataPoints[0].conditions.find(({ isDefault }) => isDefault).value;
+const getDefaultCondition = (fields) =>
+  fields[0].conditions.find(({ isDefault }) => isDefault).value;
```

Comment:
I'm thinking: we should make default the first condition in array if no `isDefault` is defined

Reply:
Definitely, it will break now. Let me fix that in next PR